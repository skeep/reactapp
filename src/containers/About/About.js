import React from 'react'
import { Helmet } from 'react-helmet'

const About = () => (
  <div>
    <Helmet>
      <title>About</title>
    </Helmet>
    <h2>About</h2>
  </div>
)

export default About
