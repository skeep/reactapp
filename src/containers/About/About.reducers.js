const AboutState = (state = {}, action) => {
  switch (action.type) {
    default:
      return state
  }
}

export default AboutState
