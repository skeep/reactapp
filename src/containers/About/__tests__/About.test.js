import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import About from '../About.js'

// Snapshot test
it('Snapshot test with content', () => {
  const tree = shallow(<About />)
  expect(toJson(tree)).toMatchSnapshot()
})

// Rendering test
it('Rendering test with content', () => {
  const tree = shallow(<About />)
  expect(tree.find('h2').text()).toEqual('About')
})

// Behaviour tests

// Integration tests

// System tests
