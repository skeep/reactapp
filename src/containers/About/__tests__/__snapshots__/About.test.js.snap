// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test with content 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      About
    </title>
  </HelmetWrapper>
  <h2>
    About
  </h2>
</div>
`;
